modules/core/src/main/java/org/apache/ignite/internal/processors/cache/ExchangeContext.java

```diff
@@ -35,6 +36,9 @@
  *
  */
 public class ExchangeContext {
+    /** Logger. */
+    private static Logger log = Logger.getLogger(ExchangeContext.class.getName());
```

Comment:
I suppose such logger initialization will lead to unexpected behavior for other logging providers (e.g. log4j). One option is to pass a logger instance as a constructor argument.

Reply:
done